src/events/interactionCreate/setParser.ts

```diff
@@ -17,11 +17,19 @@ export default event("interactionCreate", async ({ log }, interaction) => {
 	const [namespace] = readId(interaction.customId);
 	// If namespace not in setParser stop
 	if (!Object.values(Namespaces).includes(namespace)) return;
-	const parserdle = await prisma.parserdle.findFirst({
+	let parserdle = await prisma.parserdle.findFirst({
```

Comment:
Sorry that I only got to this now do we map the parsers to channel one-to-one or one-to-many?